documentation/examples/rbac-setup.yml

```diff
@@ -0,0 +1,34 @@
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRole
+metadata:
+  name: prometheus
+rules:
+- apiGroups: [""]
+  resources:
+  - nodes
+  - nodes/proxy
+  - services
+  - endpoints
+  - pods
+  verbs: ["get", "list", "watch"]
+- nonResourceURLs: ["/metrics"]
+  verbs: ["get"]
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: prometheus
+  namespace: default
```

Comment:
Question about:

> I can leave the namespace out of the ServiceAccount spec, but it's required in the ClusterRoleBinding spec

The Kubernetes docs about `ClusterRoleBinding` at https://kubernetes.io/docs/admin/authorization/rbac/#rolebinding-and-clusterrolebinding say, "a ClusterRoleBinding may be used to grant permission at the cluster level and in all namespaces.", and their example does not mention namespaces. Which part am I missing?